### PR TITLE
Add config-defaults configmap to set default values

### PR DESCRIPTION
**Pull request by savitaashture**
# Changes

Adding `config-defaults` configmap to set default values and as of now `serviceAccountName` added to config-defaults
Fixes: https://github.com/tektoncd/triggers/issues/682

/cc @dibyom @khrm 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [x] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
Triggers now support config-defaults configmap to set default values.
```

### CODE REVIEW

File: pkg/reconciler/testing/configmap.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+*/
+
+package testing
+
+import (
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	"github.com/ghodss/yaml"
```
Comment (dibyom) [inline pkg/reconciler/testing/configmap.go:24]: Can we use sigs.k8s.io/yaml instead? It is a fork of ghodss/yaml so should have the same behavior

See https://github.com/tektoncd/triggers/pull/1110#discussion_r681232522

**Review by dibyom — COMMENTED** [review]
/lgtm

File: pkg/reconciler/testing/configmap.go (hunk 1 above)
Comment (savitaashture) [reply pkg/reconciler/testing/configmap.go]: Updated :+1:

**Review by dibyom — APPROVED** [review]
/lgtm

**Pull request merged**